spring-example/pom.xml

```diff
@@ -65,32 +72,6 @@
                 <scope>test</scope>
             </dependency>
             <!--
-            The following dependencies exist solely to override Maven's default "nearest first" strategy for transitive
```

Comment:
I must admit I am not sure why these were needed, but it seems to work without them.